### PR TITLE
Add PID hypothesis in tracking

### PR DESCRIPTION
**Pull request by njacazio**
@chiarazampolli @creetz16 @shahor02

### CODE REVIEW

File: Detectors/GlobalTracking/src/MatchITSTPCQC.cxx [hunk 1]
```diff
@@ -158,6 +197,23 @@ bool MatchITSTPCQC::init()
...
+        mFractionITSTPCmatchPtVsTrkPID[i][j] = static_cast<TEfficiency*>(mFractionITSTPCmatch[i]->Clone(Form("mFractionITSTPCmatchPtVsTrkPID_%s_PID%i", title[i].c_str(), j)));
+        // Phi
+        mPhiNumVsTrkPID[i][j] = static_cast<TH1D*>(mPhiNum[i]->Clone(Form("mPhiNumVsTrkPID_%s_PID%i", title[i].c_str(), j)));
+        mPhiDenVsTrkPID[i][j] = static_cast<TH1D*>(mPhiDen[i]->Clone(Form("mPhiDenVsTrkPID_%s_PID%i", title[i].c_str(), j)));
+        mFractionITSTPCmatchPhiVsTrkPID[i][j] = static_cast<TEfficiency*>(mFractionITSTPCmatchPhi[i]->Clone(Form("mFractionITSTPCmatchPhiVsTrkPID_%s_PID%i", title[i].c_str(), j)));
+        // Eta
+        mEtaNumVsTrkPID[i][j] = static_cast<TH1D*>(mEtaNum[i]->Clone(Form("mEtaNumVsTrkPID_%s_PID%i", title[i].c_str(), j)));
+        mEtaDenVsTrkPID[i][j] = static_cast<TH1D*>(mEtaDen[i]->Clone(Form("mEtaDenVsTrkPID_%s_PID%i", title[i].c_str(), j)));
+        mFractionITSTPCmatchEtaVsTrkPID[i][j] = static_cast<TEfficiency*>(mFractionITSTPCmatchEta[i]->Clone(Form("mFractionITSTPCmatchEtaVsTrkPID_%s_PID%i", title[i].c_str(), j)));
```
Comment (shahor02) [inline Detectors/GlobalTracking/src/MatchITSTPCQC.cxx]: Thanks! Looks fine but Clone goes via disk IO, why not using constructors?

Comment (njacazio) [reply Detectors/GlobalTracking/src/MatchITSTPCQC.cxx]: True! Will do!

**Comment by njacazio** [comment]
Tested on a local MC with the changes in the QC in the linked PR

**Comment by njacazio** [comment]
@shahor02 should be good to merge

**Review by shahor02 — APPROVED** [review]

**Pull request merged**